projects/plugins/boost/app/assets/src/js/pages/getting-started/GettingStarted.svelte

```diff
@@ -0,0 +1,229 @@
+<script>
+	import { __ } from '@wordpress/i18n';
+	import ContextInfo from '../../elements/ContextInfo.svelte';
+	import ProductPrice from '../../elements/ProductPrice.svelte';
+	import TemplatedString from '../../elements/TemplatedString.svelte';
+	import Header from '../../sections/Header.svelte';
+	import config, { getStarted } from '../../stores/config';
+	import { updateModuleState } from '../../stores/modules';
+	import externalLinkTemplateVar from '../../utils/external-link-template-var';
+	import { getUpgradeURL } from '../../utils/upgrade';
+
+	$: pricing = $config.pricing;
+
+	const cssOptimizationContext = __(
+		'Move important styling information to the start of the page, which helps pages display your content sooner, so your users don’t have to wait for the entire page to load. Commonly referred to as Critical CSS.',
+		'jetpack-boost'
+	);
+
+	const deferJSContext = __(
+		'Run non-essential JavaScript after the page has loaded so that styles and images can load more quickly. Read more on <link>web.dev</link>.',
+		'jetpack-boost'
+	);
+
+	const lazyLoadingContext = __(
+		'Improve page loading speed by only loading images when they are required. Read more on <link>web.dev</link>.',
+		'jetpack-boost'
+	);
+
+	const supportContext = __(
+		`<p>Paid customers get dedicated email support from our world-class Happiness Engineers to help with any issue.</p><p>All other questions are handled by our team as quickly as we are able to go through the WordPress support forum.</p>`,
+		'jetpack-boost'
+	);
+
+	const webDevLink = 'https://web.dev/';
+
+	// svelte-ignore unused-export-let - Ignored values supplied by svelte-navigator.
+	export let navigate, location;
+	const chooseFreePlan = async () => {
+		await Promise.all( [ getStarted(), updateModuleState( 'critical-css', true ) ] );
```

Comment:
🤔 If the user chooses the paid version will it automatically enable Cloud CSS on their way back from the purchase flow?

Reply:
Yes